src/main/java/com/example/glovo/convertor/ProductDtoConverter.java

```diff
@@ -0,0 +1,16 @@
+package com.example.glovo.convertor;
+
+import com.example.glovo.dto.ProductDto;
+import com.example.glovo.model.Product;
+
+public class ProductDtoConverter {
+
+    public static Product toProduct(ProductDto productDto) {
+
+        return Product.builder()
+                .id(Product.generateId++)
```

Comment:
I don't get the point of such id for products. Can you please explain?

Reply:
I think I answered for that question above... Or you mean something else?